init-tools.sh

```diff
@@ -59,12 +57,6 @@ if [ ! -e $__PROJECT_JSON_FILE ]; then
         fi
         cd $__DOTNET_PATH
         tar -xf $__DOTNET_PATH/dotnet.tar
-        if [ -n "$BUILDTOOLS_OVERRIDE_RUNTIME" ]; then
```

Comment:
They were already broken.  The CLI stopped shipping IL only images for mscorlib.dll and System.Private.Corlib.dll so the deletion of the *.ni.dll images was going to be problematic.  In addition the paths that we were cleaning were related to the old layout of CLI, so it wasn't even copying the overridden stuff to the correct location.

There's debt to be paid down here for sure, but it's not super straightforward to do so because of the missing IL images. I want to delete the code and then fix it the right way later.